experimental/cloudstrapper/playbooks/roles/aws-cleanup/tasks/orc8r-ec2.yaml

```diff
@@ -0,0 +1,10 @@
+---
+
+- name: test to collect ec2 facts
```

Comment:
is this being used?

Reply:
Deleting of the ASG automatically takes away the EC2 instances and this file is not needed. I will remove this.